lisa/sut_orchestrator/hyperv/schema.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from dataclasses import dataclass, field
+from typing import List, Optional
+
+from dataclasses_json import dataclass_json
+
+from lisa import schema
+from lisa.util import field_metadata
+
+
+@dataclass_json
+@dataclass
+class HypervServer:
+    address: str
+    username: str
+    password: str
+
+
+@dataclass_json
+@dataclass
+class ExtraArgs:
+    command: str
+    args: str
+
+
+@dataclass_json
+@dataclass
+class SourceFileSchema:
+    source: str = field(default="", metadata=field_metadata(required=True))
```

Comment:
Because it's required, it doesn't need to provide default value.

Reply:
fixed.